src/raftstore/store/msg.rs

```diff
@@ -32,6 +32,7 @@ pub enum Tick {
     RaftLogGc,
     SplitRegionCheck,
     PdHeartbeat,
```

Comment:
PdHeatbeat chooses all leader region regularly and register the region with another timer using PdRegionHeartbeat. 
So the region heartbeat will be scattered and won't be sent in one tick.